### DISCUSSION TITLE
I don't care about cookies

### DISCUSSION THREAD

**Discussion by wolfgang-ch** [mytourbook/mytourbook#908] [General]
I've setup my Firefox browser to delete always all cookies (with exceptions) when closing the program, to get rid of the tracking cookies. But after re-opening Firefox, you always have to accept the cookies again and again.

2 Weeks ago I found this plugin https://www.i-dont-care-about-cookies.eu/ and the life is now easier again.

**Comment by FJBDev** [comment]
Good to know, I will try.
